samples/powerplay/src/main/kotlin/com/google/oboe/samples/powerplay/MainActivity.kt

```diff
@@ -285,6 +293,47 @@ class MainActivity : ComponentActivity() {
                         else -> "Performance Mode: PCM Offload"
                     }
                 )
+
+                if (OboePerformanceMode.fromInt(offload.intValue)
+                    == OboePerformanceMode.PowerSavingOffloaded && isOffloadSupported) {
+                    Spacer(modifier = Modifier.height(8.dp))
+                    Row(
+                        verticalAlignment = Alignment.CenterVertically,
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(horizontal = 32.dp)
+                            .selectable(
+                                selected = !isMMapEnabled.value,
+                                onClick = {
+                                    if (!isPlaying.value) {
+                                        isMMapEnabled.value = !isMMapEnabled.value
+                                        player.setMMapEnabled(isMMapEnabled.value)
+                                        player.teardownAudioStream()
+                                        player.setupAudioStream()
```

Comment:
Why does the MMAP toggle teardown and setup the audio stream when the performance mode toggle only tears down the audio stream?